### ISSUE TITLE
Events/human/intervention/intervention node in UN ontology?

### ISSUE THREAD

**Issue by adarshp** [clulab/eidos#527]
While playing around with CAGs, I saw that there is a node called simply "intervention" in the UN ontology - should this still be there now that there is a separate set of intervention nodes?

https://github.com/clulab/eidos/blob/2c0adc7d557371da8645ce5c738ed88f45ec936d/src/main/resources/org/clulab/wm/eidos/portuguese/ontologies/un_ontology.yml#L147

**Comment by bsharpataz** [comment]
yeah..... i have mixed feelings..... but it's a longer discussion than i
feel like having here :) tomorrow all together?
